docs/quickstarts/index.rst

```diff
@@ -6,4 +6,3 @@ Quickstarts
    :caption: Contents:
 
    use-events-to-call-webhook
-   event-bus
```

Comment:
@mariajgrimaldi: What happened to the event-bus doc? Did you move it to a new branch, or just kill it?

Reply:
Please ignore. I was looking at a diff, and not the full PR.